### ISSUE TITLE
Request accessibility/508 review for GIBCT Colmery 107 updates (branches & extensions)

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#1544]
## Team Name

BAH

## Product Description

As a user of the Comparison Tool, I need the Comparison Tool on VA.gov to support the legislative changes for Section 107 of the Forever GI Bill so that I am able to view informaiton that calculates housing based on location of the campus where the majority of my classes are taken.

## Staging URL and Credentials

staging.va.gov/gi-bill-comparison-tool/

## Test Plan

[CT_107_Accessibility Testing.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/3576249/CT_107_Accessibility.Testing.xlsx)

## Known Issues

- None to report

[508 Issue Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=)

- **Related Issue URL:** _Delete if no known issues._

**Comment by desireeturner** [comment]
@1Copenut We're ready for you to perform your accessibility review of our updates to the GI Bill Comparison Tool around Colmery Section 107. Once you've had a chance to review this request, can you provide us with a rough estimate on when you'll have time to complete it? Thank you!

**Comment by 1Copenut** [comment]
@desireeturner Thank you for the confirmation. I'll give it a look tomorrow and get you an estimate of the timeline to complete.

**Comment by 1Copenut** [comment]
@desireeturner Looking at the test plan and staging v. production, I should be done auditing by EOD Tuesday, September 10th. Will advise if that date changes plus or minus.

**Comment by 1Copenut** [comment]
508 testing completed so far:
* Axe/ATTEST review - DONE
* Keyboard review - DONE
* Zoom review - IN PROGRESS
* Screenreader review - TODO
* Screen zoom - TODO

**Comment by 1Copenut** [comment]
@desireeturner && @andreahewitt-odd I've finished the 508 pre-launch audit and opened an epic with six issues for the findings: https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/1639

I'm moving this request ticket to close and adding a row to our Product 508 sheet for tracking purposes. I will follow up in a week to see how issues are progressing and coordinate a quick test with the VA 508 office.
